infra/build/developer-tools/build/scripts/module-swapper/cmd/testdata/example-module-with-submodules/examples/example-one/main.tf.good

```diff
@@ -0,0 +1,24 @@
+module "test-module" {
+  source       = "../.."# [restore-marker]   source       = "terraform-google-modules/example-module-with-submodules/google"
+# [restore-marker]   version      = "~> 3.2.0"
```

Comment:
Will this break linting? We should try to pass linters.

Reply:
Because we use `fmtTF` when possible, this fixes some whitespace issues for us. I have tested with `terraform-google-cloud-router` and that passes linters.